concepts/modularization/README.md

```diff
@@ -41,13 +41,47 @@ If the API you need (like a core Kubernetes API or Istio virtual service) is not
 # Release channels
 Release channels let customers try new modules and features early, and decide when the updates should be applied. 
 
+The first use case will be modeled as the alpha channel. Modules available in the alpha channel are developed with all quality measures in place (functional correctness, security, etc.), but they might still have unstable API or be changed without keeping backward compatibility. When you use a module from the alpha channel, you won't get full SLA guarantees for that module or other modules that are affected (directly or indirectly).
```

Comment:
I think we should add that dependencies can break and there is no guarantee on the removal of a module at a later point (even if its already implicitly said as its not part of SLAs)